src/Tools/dotnet-monitor/CollectionRules/Options/CollectionRuleDefaultsOptions.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Diagnostics.Monitoring.WebApi;
+using System.ComponentModel.DataAnnotations;
+
+namespace Microsoft.Diagnostics.Tools.Monitor.CollectionRules.Options
+{
+    internal class CollectionRuleDefaultsOptions
+    {
+        [Display(
+            ResourceType = typeof(OptionsDisplayStrings),
+            Description = nameof(OptionsDisplayStrings.DisplayAttributeDescription_CollectionRuleDefaultsOptions_Triggers))]
+        public CollectionRuleTriggerDefaultsOptions Triggers { get; set; } = new();
```

Comment:
I don't think the `= new();` on these properties is necessary. It might make it more difficult to determine if these properties were actually set by the user rather than happen to be filled by deserialization.